### ISSUE TITLE
dynatrace_disk_anomalies with metric LowDiskSpace fails.

### ISSUE THREAD

**Issue by JosephHobbs** [dynatrace-oss/terraform-provider-dynatrace#78]
While trying to create a new resource **dynatrace_disk_anomalies** with metric **LowDiskSpace**, I receive the following error:

![image](https://user-images.githubusercontent.com/43007348/134789760-6a81be23-2d59-4b51-8132-aec9ce79d802.png)

If I use the metric name **LOW_DISK_SPACE** it works as expected.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
@JosephHobbs can you share a HCL snippet with me?

Local tests with this example here seem to work fine.

```resource "dynatrace_disk_anomalies" "foobar" {
  name              = "foobar"
  enabled           = true
  metric            = "LOW_DISK_SPACE"
  samples           = 5
  threshold         = 10
  violating_samples = 5
  disk_name {
    operator = "CONTAINS"
    value    = "888"
  }
}
```

**Comment by JosephHobbs** [comment]
I think the confusion is with the documentation. It looks like the metric is named LowDiskSpace vs LOW_DISK_SPACE.

![Screenshot_20210930-235023~2](https://user-images.githubusercontent.com/43007348/135562881-77ca890e-055b-4b7b-b144-607de06a3959.png)

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Agreed, using the documentation only this pitfall is inevitable. I'll update the documentation and think about ways to make this more comfortable to configure.

**Comment by JosephHobbs** [comment]
Honestly, I think just including the metrics in the documentation would be sufficient to keep users on track.  I went ahead and submitted a PR for the docs update.  I may not have a handle on Go yet, but I can write documentation!

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hi @JosephHobbs,

Your Pull Request has been merged into [v1.80](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.8.0).

Thanks for contributing - much appreciated.
